microsite/pages/en/plugins.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const fs = require('fs');
+const yaml = require('js-yaml');
+const React = require('react');
+const Components = require(`${process.cwd()}/core/Components.js`);
+const {
+  Block: { Container },
+  BulletLine,
+} = Components;
+
+const pluginsDirectory = require('path').join(process.cwd(), 'data/plugins');
+const pluginMetadata = fs
+  .readdirSync(pluginsDirectory)
+  .map(file =>
+    yaml.safeLoad(fs.readFileSync(`./data/plugins/${file}`, 'utf8')),
+  );
+const truncate = text =>
+  text.length > 170 ? text.substr(0, 170) + '...' : text;
+
+const addPluginDocsLink =
+  'http://localhost:3000/docs/plugins/add-to-marketplace';
```

Comment:
Can't this be `/docs/plugins/add-to-marketplace`

Reply:
😬 Thanks!